common/src/main/java/com/wynntils/functions/WorldFunctions.java

```diff
@@ -58,6 +60,129 @@ public List<String> getAliases() {
         }
     }
 
+    public static class TokenGatekeeperCountFunction extends Function<Integer> {
+        @Override
+        public Integer getValue(FunctionArguments arguments) {
+            return Models.Token.getGatekeepers().size();
+        }
+
+        @Override
+        public List<String> getAliases() {
+            return List.of("token_count");
+        }
+    }
+
+    public static class TokenGatekeeperDepositedFunction extends Function<CappedValue> {
+        @Override
+        public CappedValue getValue(FunctionArguments arguments) {
+            List<TokenGatekeeper> gatekeepers = Models.Token.getGatekeepers();
+            if (gatekeepers.isEmpty()) return CappedValue.EMPTY;
+
+            return gatekeepers.get(0).getDeposited();
```

Comment:
Why don't these take an argument for index, like other functions (others are required, but you could make this default 0 if you want)?

Reply:
Hm, I did not think of that...

Reply:
There are two sets of all functions, one which defaults to index 0, and one which takes an index argument. But you are right that having 0 as default would be better. I'll fix.